executor/aggregate.go

```diff
@@ -406,8 +425,19 @@ func (w *HashAggPartialWorker) run(ctx sessionctx.Context, waitGroup *sync.WaitG
 	}
 }
 
+func getGroupKeyMemUsage(groupKey [][]byte) int64 {
```

Comment:
Can we move the calculation in `getGroupKey` instead of calculating it again?

Reply:
We can't... w.groupKeys will also be update in L584. It doesn't call the funtion getGroupKey.